module.json

```diff
@@ -9,9 +9,9 @@
         "organisation",
         "sharing"
     ],
-    "version": "0.12.1",
+    "version": "0.13.0",
     "humhub": {
-        "minVersion": "1.7"
+        "minVersion": "1.8"
```

Comment:
Is this change no longer compatible with 1.7?

Reply:
ah yes, the change doesn't break 1.7, then as I understood we should keep 1.7 here and in all other similar changes in other modules.

Reply:
ok I reverted min version to 1.7. in all external modules where I did the change today.

Reply:
Great! Thanks!